toolchain/syno-aarch64-7.0/Makefile

```diff
@@ -0,0 +1,16 @@
+TC_NAME = syno-aarch64
+
+TC_ARCH = rtd1296 armada37xx aarch64
+TC_VERS = 7.0
+TC_KERNEL = 4.4.180
+TC_GCC = 7.3.0
+TC_GLIBC = 2.26
+
+TC_DIST = rtd1296-gcc730_glibc226_armv8-GPL
+TC_DIST_SITE_PATH = Realtek%20RTD129x%20Linux%204.4.180
+TC_DIST_SITE_URL = https://sourceforge.net/projects/dsgpl/files/Tool%20Chain/DSM%207.0%20preview%20Tool%20Chains/
```

Comment:
An option here could be to change `spksrc.tc-flags.mk` such as:
```
ifeq ($(strip $(TC_DIST_SITE_URL)),)
ifeq ($(TC_VERS),7.0)
TC_DIST_SITE_URL = https://sourceforge.net/projects/dsgpl/files/Tool%20Chain/$(TC_TYPE)%20$(TC_VERS)%20preview%20Tool%20Chains/
endif
TC_DIST_SITE_URL = https://sourceforge.net/projects/dsgpl/files/Tool%20Chain/$(TC_TYPE)%20$(TC_VERS)%20Tool%20Chains/
endif
```
And therefore get rid of all the unnecessary duplicate line from each toolchain definitions.

Reply:
That's a good point. I would just wait until Synology releases the toolchains for the released version. Until then I don't see an issue having them there. It's in beta after all

Reply:
Indeed but migration path afterwards become less cumbersome by changing a single file instead of +++